contrib/auto-render/test/auto-render-spec.js

```diff
@@ -146,6 +145,19 @@ describe("A delimiter splitter", function() {
             ]);
     });
 
+    it("correctly processes sequences of $..$", function() {
+        expect("$hello$$world$$boo$").toSplitInto(
```

Comment:
There should be a test case where both `$` and `$$ are detected correctly, e.g. `"$hello$world$$boo$$"` or `"$hello$$world$$$boo$$"` or something like that.